ui/components/config/provider-form.tsx

```
@@ -916,6 +920,21 @@ export default function ProviderForm({ provider, onSave, onCancel, existingProvi
                           />
                         </div>
                       </div>
+
+                      <div className="mt-6 space-y-4">
+                        <div className="flex items-center justify-between space-x-2">
+                          <div className="space-y-0.5">
+                            <label className="text-sm font-medium">Include Raw Response</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)